SimpleWAWS/Controllers/ResourceController.cs

```diff
@@ -258,5 +266,9 @@ public Task<HttpResponseMessage> All()
                 return Request.CreateResponse(HttpStatusCode.InternalServerError, Resources.Server.Error_GeneralErrorMessage);
             }
         }
+        public  HttpResponseMessage GetUserIdentityName()
```

Comment:
Just something I added for debugging. Sometimes my id was something like MSA#user@hotmail.com other times  live.com#user@hotmail.com  . Just so I know what to put int he admin whitelist.